.travis.yml

```diff
@@ -10,6 +10,8 @@ addons:
     - libacl1-dev
     - libacl1
     - build-essential
+    - python3-dbus
```

Comment:
Good intention, but the test happens in pyenv-supplied Python, not the system Python (which is too old). I'm trying to install dbus there. If it doesn't work, we'll just mock it.

Reply:
Looks like Travis' dbus is too old anyways.

`Requested 'dbus-1 >= 1.8' but version of dbus is 1.6.18`